pkg/image/importer/importer.go

```diff
@@ -282,6 +282,27 @@ func applyErrorToRepository(repository *importRepository, err error) {
 	}
 }
 
+func calculateImageSize(ctx gocontext.Context, repo distribution.Repository, image *api.Image) error {
+	bs := repo.Blobs(ctx)
+
+	layerSet := sets.NewString()
+	size := int64(0)
+	for i := range image.DockerImageLayers {
+		layer := &image.DockerImageLayers[i]
+		desc, err := bs.Stat(ctx, digest.Digest(layer.Name))
```

Comment:
It is acceptable if in the master branch of Origin we explicitly call out that importing schema1 from the DockerHub (that have been down converted from schema2) will not have size set, because we anticipate this problem resolving itself by a) more images on the hub being schema2, b) we want to support schema2 import on the rebase.

I'll open a PR to disable the tests now and we can lower the priority.